catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -59,55 +60,72 @@
 
     private int subDirLevels;
 
+    private PeriodicBatchExecutor<Metacard> periodicDrain;
+
     private boolean enableBackupPlugin = true;
 
     public CatalogBackupPlugin() {
+
+        init();
+    }
+
+    private void init() {
```

Comment:
I'd make this method `public` and have blueprint call it using `init-method` instead of calling it from the constructor.